ata_pipeline0/main.py

```diff
@@ -24,14 +24,27 @@
 
 def handler(event, context):
     # Note: this is invoked by an event-driven, async method (s3 trigger) so the return value is discarded
-    print("request: {}".format(json.dumps(event)))
-    return {"hi": "there"}
+    bucket_name = event["Records"][0]["s3"]["bucket"]["name"]
```

Comment:
❓ Would it be possible to see what the `event` dictionary looks like? A dummy example will work just fine!

Reply:
Yup I'll link to an example from their docs in a comment